<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>511</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <property name="allColumnsShowFocus">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CommandLinkButton</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Торгово-арбитражный робот Форсаж v1.0&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:10pt; color:#000000;&quot;&gt;GUI-интерфейс Торгово-арбитражный робот Форсаж, версия 1.0 от 13.06.2018&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:10pt; color:#000000;&quot;&gt;Авторские права © Минковский Никита&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:10pt; color:#000000;&quot;&gt;ПО 'Торгово-арбитражный робот Форсаж' предназначено для проведения торгов на онлайн биржах&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-size:10pt; color:#000000;&quot;&gt;Применение ПО избавляет от человеческого фактора при работа с онлайн биржами.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Сигнал продажи</string>
     </property>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Сигнал покупки</string>
     </property>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Еще...</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_3">
   <property name="text">
    <string>Просмотр логов</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Включить</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Включить</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Настройки программы</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
